modules/openapi-generator/src/main/resources/JavaSpring/api.mustache

```diff
@@ -196,16 +196,16 @@ public interface {{classname}} {
         responseContainer = "{{{.}}}"{{/returnContainer}}{{#hasAuthMethods}},
         authorizations = {
         {{#authMethods}}
-        {{#isOAuth}}
+        {{#scopes.0}}
```

Comment:
Hi welshm, Thanks for your review.
This is idiom that checks if the array `scopes` is not empty, which is already used in the many places of existing templates (ex: [here](https://github.com/kota65535/openapi-generator/blob/d9920a5b819c8350d2051a5a066b234398b7db76/modules/openapi-generator/src/main/resources/Java/libraries/apache-httpclient/api.mustache#L9C11-L9C11), and [here](https://github.com/kota65535/openapi-generator/blob/d9920a5b819c8350d2051a5a066b234398b7db76/modules/openapi-generator/src/main/resources/php/libraries/psr-18/api.mustache#L699)).

So when multiple scopes are specified...
```yaml
security:
  - bearer: [ foo, bar ]
```

It will be generated as follows:
```java
security = {
    @SecurityRequirement(name = "bearer", scopes={ "foo", "bar" })
}
```